packages/manager/apps/web/client/app/dns-zone/zone.routing.js

```diff
@@ -23,6 +21,52 @@ export default /* @ngInject */ ($stateProvider) => {
       defaultFilterColumn: () => 'name',
       header: /* @ngInject */ ($translate) => $translate.instant('zones_title'),
       customizableColumns: () => true,
+      columns: /* @ngInject */ ($translate) => [
+        {
+          property: 'name',
+          title: $translate.instant('zones_domain_name'),
+          template: `<a data-ng-href="{{ $ctrl.getServiceNameLink($row) }}" data-ng-bind="$row.name"></a>`,
+          searchable: true,
+          sortable: 'asc',
+          filterable: true,
+        },
+        {
+          property: 'hasDnsAnycast',
+          title: 'DNS Anycast',
```

Comment:
**question:** Dont we need translation for this?

Reply:
Hello @anooparveti ,
No, `DNS Anycast` and `DNSSEC` are considered generic and therefore do not need to be translated.